### ISSUE TITLE
Tex does not compile

### ISSUE THREAD

**Issue by msmathcomp** [ofloveandhate/git_training#2]
When I downloaded your tex, it gave me an error when compiling it:

"File `UWEC-stacked-wordmark-Blue_cmyk' not found. \end{frame}"

**Comment by ofloveandhate** [comment]
yes, i know...  those images are not mine to share.  please comment out that line and try again.

**Comment by rantonse** [comment]
Hey, can I join?
